### PR TITLE
Five Pillars Of Skycoin.it.md

### PR DESCRIPTION
**Pull request by edoardocoen**
cTjj1VLdiA1A2xHWqkXxmJXZWAc3SVmbzo

### CODE REVIEW

File: content/statement/Five Pillars Of Skycoin.it.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+(2) [Skywire](/tags/skywire/), una rete a maglie, in cui è possibile ottenere monete inoltrando 
+   traffico/larghezza di banda. Questo è simile a Tor o BitTorrent,
+   a differenza del fatto che verrai pagato per eseguire un nodo.
+
+(3) [Skyledger](https://www.skyledger.net), una piattaforma che è la nostra versione del 
+   ERC20 di Ethereum, ma migliore:
+
+ - Diamo ad ogni valuta la propria catena di blocchi, la piattaforma gira sull'infrastruttura 
+   Skywire e riceviamo una commissione per questo.
```
Comment (focamonca) [inline content/statement/Five Pillars Of Skycoin.it.md]: sei andato a capo spezzando la frase

File: content/statement/Five Pillars Of Skycoin.it.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ - Abbiamo un linguaggio di programmazione ([CX](/overview/cx-overview/)),
+   non "contratti intelligenti" (che pensiamo siano spazzatura).
```
Comment (focamonca) [inline content/statement/Five Pillars Of Skycoin.it.md]: lascia *smart contracts*

File: content/statement/Five Pillars Of Skycoin.it.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ - Permettiamo agli sviluppatori di ciascuna catena di codificare tutto ciò che vogliono.
+ 
+(4) Un ecosistema di "app" costruito all'interno delle piattaforme precedenti:
```
Comment (focamonca) [inline content/statement/Five Pillars Of Skycoin.it.md]: non mi suona benissimo

**Comment by gz-c** [comment]
conflicting file?

**Comment by edoardocoen** [comment]
I can't solve the conflict. I close this pull request and I create new one

**Pull request closed without merging**